doc/nrf/device_guides/working_with_nrf/nrf91/nrf9160_gs.rst

```diff
@@ -43,83 +43,33 @@ On your computer, one of the following operating systems:
 |Supported OS|
 
 .. _nrf9160_gs_installing_software:
-
-Installing the required software
-********************************
-
-To work with the nRF9160 DK firmware and certificates, install `nRF Connect for Desktop`_.
-
-After installing and starting the application, install the `Cellular Monitor`_ app.
```

Comment:
We mention about cellular monitor in the [provisioning ](https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-12900/nrf/device_guides/working_with_nrf/nrf91/nrf9160.html#provisioning-the-nrf-cloud-certificate) section. The steps could be updated there to install the `Cellular Monitor`_ app, and then open the app, since these steps are now removed?